### ISSUE TITLE
Project continued 

### ISSUE THREAD

**Issue by The-Code-Monkey** [styled-system/styled-system#2162]
Hey guys, I have continued this project in my own repo and seeing as people keep asking for it. I have been keeping it up to date for at least 6 months.

https://github.com/The-Code-Monkey/TechStack/tree/dev/packages/styled-system

It works exactly like this version. There are currently no docs because you can refer to the ones in this repo.

I am also trying to get a hold of @jxnblk with no luck so if anyone has any way to contact him that would be useful

To install the updated version `yarn add @teckstack/styled-system`

**Comment by nstanard** [comment]
@The-Code-Monkey Might be good to create a dedicated repo for your version though if you plan on keeping it up and allowing other contributors.

**Comment by The-Code-Monkey** [comment]
Yeah might do that actually as it would mean more people would find it

**Comment by The-Code-Monkey** [comment]
@nstanard here is a link to my dedicated repo for styled system. 

https://github.com/The-Code-Monkey/styled-system

**Comment by AS-EBaiden** [comment]
So, what happened? why is this not maintained on here anymore?

**Comment by The-Code-Monkey** [comment]
The maintainer moved on to other projects and hasn't replied to my request for access to the repo so I moved it to my own

**Comment by bxt** [comment]
Some alternatives and discussion can be found here: https://github.com/styled-system/styled-system/issues/1547

**Comment by Kikobeats** [comment]
@The-Code-Monkey seems your project is archived ☹️

**Comment by The-Code-Monkey** [comment]
Unfortunately due to work and other commitments I'm unable to continue feel
free to fork and update the project or if you want I can unarchive it and
add you as a contributor?

**Comment by Kikobeats** [comment]
tbh your fork is working fine to me – let me ping you if I find some room for improvement 🙂
